README.md

```diff
@@ -3,3 +3,10 @@
 This is a package with syntax highlighter for EdgeDB languages: EdgeQL and
 ESDL (EdgeDB Schema Definition Language).  The plugin is designed to work with
 vim and neovim.
+
+## Usage
+Copy this repo in your vim pack directory. More info in [Vim docs](https://github.com/vim/vim/blob/247bf0de465411e4ebb1c0fe1a9e07f4a7f77e91/runtime/doc/repeat.txt#L515)
```

Comment:
There are actually two ways of installing edgedb-vim:

1. Use https://github.com/VundleVim/Vundle.vim or https://github.com/junegunn/vim-plug

2. Manual clone

Reply:
@1st1 I added both now.